driver/razermouse_driver.c

```diff
@@ -1589,6 +1635,29 @@ static ssize_t razer_attr_write_scroll_mode_none(struct device *dev, struct devi
     return count;
 }
 
+/**
+ * Write device file "mode_wave" (for extended mouse matrix effects)
```

Comment:
Please fix the comment

Reply:
I've fixed the comment here along with other comments to reference scroll.